recipe/meta.yaml

```
@@ -57,18 +56,23 @@ requirements:
     - click >=7.0,<9
     - cloudpickle <4
     - databricks-sdk >=0.20.0,<1
+    - fastapi <1
     - gitpython >=3.1.9,<4
     - importlib-metadata <9,>=3.7.0,!=4.7.0
     - opentelemetry-api >=1.9.0,<3
     - opentelemetry-sdk >=1.9.0,<3
     - packaging <25
+    - pydantic >=1.10.8,<3
+    - typing-extensions >=4.0.0,<5
+    - uvicorn <1
     # required on imports for win
     - pytz <2025  # [win]
     - protobuf >=3.12.0,<6
```

```suggestion
    - protobuf >=3.12.0,<7
```
https://github.com/mlflow/mlflow/blob/v3.3.2/libs/skinny/pyproject.toml#L38